### DISCUSSION TITLE
6/12일 반성문

### DISCUSSION THREAD

**Discussion by shkisme** [BDD-CLUB/challenge-every-week-posting#68] [6. 반성문]
저번 주 꾸글쓰 글을 업로드하지 못했습니다 ㅠㅠ
사실 생각하고 있는 주제는 꽤 많았는데요. (키퍼 홈페이지 리뉴얼2 개발 정리, 이펙티브 자바 정리, 개발 공부 방향성에 대한 고찰 등등등등...) 그런데 글로 정리하려니 생각 정리가 잘 안되고 막막하여 글을 업로드하지 못했습니다. 요새 신경쓰고 있는 부분이 많다보니 그런 것 같아요. 시험기간이기도 하고 ㅎㅎ

반성문을 이렇게 써보니 다시 한번 다짐을 하게 되네요! 열심히 해보겠습니다!

**Comment by publdaze** [comment]
화이팅화이팅!

**Comment by lcqff** [comment]
홧팅홧팅~~~!
